### PR TITLE
Export remaining CSV fields

### PR DESCRIPTION
**Pull request by tingaloo**
I rule and have completed some work on Case Manager that's ready for review!

This PR makes the assumption that the first and last call are patient.calls are indeed the first and last.

This pull request makes the following changes:
* Exports call and fulfillment information

It relates to the following issue #s: 
* Bumps #965

### CODE REVIEW

**Comment by lomky** [comment]
Note to self for later in #812 - export needs to function on both active & archived patients

**Comment by lomky** [comment]
I glanced at the circleCI failure, and I'm pretty sure it's a false positive since you didn't touch that part of the code.

**Comment by colinxfleming** [comment]
Merging once ci clears! Thanks @tingaloo and @lomky !

**Comment by colinxfleming** [comment]
I'm not sure what's up but I think this might be causing legit test failures -- I'm consistently getting errors on the following:

```
 FAIL["test_0003_should log a new patient ready for further editing: abortion", #<Class:0x007ff7f23d7a38>, 72.98456285100838]
 test_0003_should log a new patient ready for further editing: abortion#DataEntryTest::entering a new patient (72.99s)
        Expected false to be truthy.
        test/integration/data_entry_test.rb:92:in `block (3 levels) in <class:DataEntryTest>'
        test/integration/data_entry_test.rb:90:in `block (2 levels) in <class:DataEntryTest>'

ERROR["test_0001_should update balance on field change", #<Class:0x007ff7f4d66598>, 172.1067891200073]
 test_0001_should update balance on field change#UpdatePatientInfoTest::changing abortion information (172.11s)
Capybara::ElementNotFound:         Capybara::ElementNotFound: Unable to find css "#outstanding-balance"
            test/integration/update_patient_info_test.rb:72:in `block (2 levels) in <class:UpdatePatientInfoTest>'

 FAIL["test_0002_should alter the information", #<Class:0x007ff7f4d66598>, 182.13227110001026]
 test_0002_should alter the information#UpdatePatientInfoTest::changing abortion information (182.13s)
        Expected false to be truthy.
        test/integration/update_patient_info_test.rb:93:in `block (3 levels) in <class:UpdatePatientInfoTest>'
        test/integration/update_patient_info_test.rb:87:in `block (2 levels) in <class:UpdatePatientInfoTest>'
```

@tingaloo can you dig in and see what's up here?

**Comment by tingaloo** [comment]
I'll look into this later, I'm guessing I need to :try each field because I'm assuming patients have at least 1 call.

**Comment by tingaloo** [comment]
The tests should be fine now. I think ```procedure_cost``` fails because it is an integer. I think I have to cast it to an integer but it is a little weird with the try methods floating around.

File: app/models/concerns/exportable.rb [hunk 1]
```diff
@@ -36,15 +36,66 @@ module Exportable
...
+    "Pledge generated time" => :pledge_generated_at,
+
+    "Timestamp of first call" => :first_call_timestamp,
+    "Timestamp of last call" => :last_call_timestamp,
+    "Call count" => :call_count,
+    "Reached Patient call count" => :reached_patient_call_count,
+
+    "Fulfilled" => :fulfilled,
+    "Procedure date" => :procedure_date,
+    "Gestation at Procedure" => :gestation_at_procedure,
```
Comment (colinxfleming) [inline app/models/concerns/exportable.rb]: We'll want to clarify that this is in weeks. Right now it's returned as an int which isn't terribly clear. I'll fix it up.

**Pull request merged**